### DISCUSSION TITLE
General Availability

### DISCUSSION THREAD

**Discussion by pmcoder01** [amazonlinux/amazon-linux-2022#131] [General]
Is there any estimation for the GA date yet? AWS supports said it will be around Q4 2022, I hope it is sooner!

**Comment by ohkjames** [comment]
Asking for the same question, we have some outdated Amazon Linux 1 that needs to be upgraded. It would be a waste of effort to go to AWS Linux 2 when 2022 is around the corner.

Do we have any estimates? Some time in Q1 2023?

**Comment by jeroenhabets** [comment]
> Do we have any estimates? Some time in Q1 2023?

Don't hold your breath for Q1 though the name change to 2023 gives solid hope to a release somewhere this year (2023).

Good news is that on the [Amazon Linux 2023 Feature Backlog ](https://github.com/amazonlinux/amazon-linux-2023/projects/1) the important [[Feature Request] - Kernel Live Patching](https://github.com/amazonlinux/amazon-linux-2023/issues/5) has just been Closed by @stewartsmith .

Not sure if the other three Open there need to ship before GA or not. 

@stewartsmith, @nmeyerhans IMHO It would be helpful if this project had a "2023 GA" Milestone (and at least one for beyond) to attach to both roadmap items and issues so it'd be clear where we stand. Would you agree?

P.S. We're also planning a OS upgrade so a we're a bit in the same boat as @ohkjames [comment](https://github.com/amazonlinux/amazon-linux-2023/discussions/131#discussioncomment-3849083)

**Comment by kemotaha** [comment]
We have a definite date for GA, and it is today: https://aws.amazon.com/about-aws/whats-new/2023/03/amazon-linux-2023/

**Comment by pmcoder01** [comment]
Well done guys! Thank you 👏🏻👏🏻👏🏻👌🏻

**Comment by jeroenhabets** [comment]
Congratulations to you @Kemotaha and the whole team 🚀 👏 🙇 !
